src/main/java/fact/extraction/WaveformFluctuation.java

```diff
@@ -56,17 +56,7 @@ public Data process(Data input) {
         Utils.isKeyValid(input, "NPIX", Integer.class);
         npix = (Integer) input.get("NPIX");
 
-        int[] pixels = null;
-
-        //Load a given pixelset, otherwise use the the whole camera
-        if (input.containsKey(pixelSetKey)) {
-            Utils.isKeyValid(input, pixelSetKey, PixelSet.class);
-            PixelSet pixelSet = (PixelSet) input.get(pixelSetKey);
-            pixels = pixelSet.toIntArray();
-        } else {
-            ContiguousSet<Integer> numbers = ContiguousSet.create(Range.closed(0, npix-1), DiscreteDomain.integers());
-            pixels = Ints.toArray(numbers);
-        }
```

Comment:
This leads to the behavior that a typo in the pixelset results in the processor processing the whole camera. I think this should rather throw an error.